scripts/automation/docker/deploy-hawkular.sh

```diff
@@ -88,17 +88,23 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
-${OC_} get projects | grep ${OS_PROJECT_}
+${OC_} get projects | grep "${OS_PROJECT_}[[:space:]]"
 if [ $? -eq 0 ]; then
     echo "Deleting project:${OS_PROJECT_}"
-    ${OC_} delete project ${OS_PROJECT_}
-    if [ $? -ne 0 ]; then
+    OC_RESULT=`${OC_} delete project ${OS_PROJECT_}`
+    if [ $? -ne 0 ] ; then
         echo "Failed to delete project: ${OS_PROJECT_}"
-        exit 1
+        echo ${OC_RESULT} | grep --quiet "The system is ensuring all content is removed from this namespace"
+        if [ $? = 1 ]
+        then
+          echo "Project ${OS_PROJECT_} is already terminating"
+        else
+          exit 1
+        fi
     fi
 
     while true; do
-        ${OC_} get projects | grep ${OS_PROJECT_}
+        ${OC_} get projects | grep "${OS_PROJECT_}[[:space:]]"
```

Comment:
In case of more projects with similar names g.e. vprusa-hs & vprusa-hs2 this will grep just the shorter one ( vprusa-hs )